src/components/PostList/PostList.jsx

```diff
@@ -1 +1,12 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo';
+import commentsFromServer from '../../api/comments.json';
+import usersFromServer from '../../api/users.json';
+
+function getCommentByPostId(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId);
+}
+
+export const PostList = ({ posts }) => (
```

Comment:
something goes wrong here. 
1. You should create a variable at App
```
const posts = postsFromServer.map(post => (
  {
    ...post,
    user: usersFromServer.find(user => user.id === post.userId),
    comments: commentsFromServer.filter(comment => comment.postId === post.id),
  }
));
```

2. pass this variable to PostList
```
    <PostList posts={posts} />

```

3. at PostList you should accept posts, map them and render `PostInfo`

Reply:
* NOT FIXED *